images/ci/Dockerfile

```diff
@@ -5,5 +5,4 @@ RUN make build --warn-undefined-variables
 
 FROM registry.ci.openshift.org/ocp/4.9:base
 COPY --from=builder /go/src/github.com/openshift/cert-manager-operator/cert-manager-operator /usr/bin/
-COPY manifests /manifests
 LABEL io.openshift.release.operator=true
```

Comment:
not sure about this one i think we should remove it as well.

Reply:
https://github.com/openshift/oc/blob/f94afb52dc8a3185b3b9eacaf92ec34d80f8708d/pkg/cli/admin/release/annotations.go#L9-L11

Reply:
https://github.com/openshift/oc/blob/a765590e1b87b014b9d81f9ea534460d6dff73f2/pkg/cli/admin/release/new.go#L73-L82

Reply:
Removed.